### DISCUSSION TITLE
German Translation

### DISCUSSION THREAD

**Discussion by roma-marshall** [vuejs-translations/guidelines#49] [Q&A] [answered]
## Hi there!

I started to translate Vue 3 documentation in German.
This will help to learn the framework for a German speaking audience.
Anyone who would like to help me write in the comments.
Below I will post guidelines how to help me with the translation.

### How to  participate:

If you are interested in participating in the translation process,
you should make a fork of the documentation repository that is in my GitHub profile,
then translate any part of the documentation, create a pull request, and assign it to me.

Page: https://de.vuejs.org
Repo: https://github.com/vuejs-translations/docs-de

**Comment by roma-marshall** [comment, marked as answer]
# Guidelines:

> ### 1. Fork [this repository](https://github.com/vuejs-translations/docs-de)
> ### 2. Translate some part of documentation
> ### 3. Create a new PR(Pull Request) and assign it to me @roma-marshall

**Comment by Marcys** [comment]
I would love to contribute :)

**Reply by roma-marshall** [reply]
You are welcome! Just fork [**repository**](https://github.com/roma-marshall/docs-de), translate some part of docs and send me PR.

**Comment by hwesselmann** [comment]
Hi! I also like to contribute and added a PR for the page on conditional rendering. I have not translated word-by-word, but tried to get a fluent reading experience while keeping all the content correct :-)

**Reply by roma-marshall** [reply]
Good Job! Thank you for your contribution!

**Reply by hwesselmann** [reply]
Thanks! I will try to keep on with translating and will create a PR once a section ist ready to be reviewed :-)

**Comment by aubcel** [comment]
Hey!
Just saw that you're still working on it. 🎉 
Is there a way I could support you?

**Reply by roma-marshall** [reply]
Hey, sure! Just fork [this repository](https://github.com/roma-marshall/docs-de) and create a PR with your translations.

**Comment by roma-marshall** [comment]
hello @kiaking, @Jinjiang and @yyx990803. I would like to transfer the [german repo](https://github.com/roma-marshall/docs-de) into the [vuejs-translations](https://github.com/vuejs-translations) organization. Please let me know when we can do this and what is our next step.

**Reply by Jinjiang** [reply]
@roma-marshall 

I'd like to check things below before the transfer, according to our [guidelines](https://github.com/vuejs-translations/guidelines):

1. have you completed more than 50% of the content translated?
2. please [detach your fork](https://support.github.com/request/fork).

Thanks.

**Reply by roma-marshall** [reply]
@Jinjiang

1. Yes, we did about 50% or more.
2. I have detached the fork for you now: https://github.com/roma-marshall/docs-de

**Reply by Jinjiang** [reply]
Great! I've sent an invitation to you to join the org. After accepting that you will be able to do the transfer. Thanks.

**Comment by Hossaini1** [comment]
Hello, I would like to contribute to the translation.

**Reply by roma-marshall** [reply]
Great, take a look at the [guideline]( https://github.com/orgs/vuejs-translations/discussions/49#discussioncomment-4149265)

**Reply by Hossaini1** [reply]
ok thanks

**Comment by bcostaaa01** [comment]
Hallo, ich könnte auch ein bisschen helfen! I also help here? :=)

Is there a list of currently undergoing translated pages or parts of the docs, or do I just open a PR and pick a part of the docs?

**Reply by roma-marshall** [reply]
Hello, just pick any English doc, translate it to German, open a Pull Request, and tag me.
